src/main/java/frc/robot/subsystems/HoodSubsystem.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.LimitSwitchNormal;
+import com.ctre.phoenix.motorcontrol.LimitSwitchSource;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import frc.robot.RobotConstants;
+import frc.robot.controllers.TalonPositionController;
+import frc.robot.utils.ClampedDouble;
+
+/**
+ *
+ * @author dcowden
+ */
+public class HoodSubsystem extends BaseSubsystem {
+
+    private WPI_TalonSRX hoodMotor;
+    private TalonPositionController hoodMotorController;
+    private final PositionAngleConverter positionConverter = new PositionAngleConverter();
+    public static final double HOOD_TOLERANCE_COUNTS = 50;
+
+    private final ClampedDouble desiredHoodPositionDegrees = ClampedDouble.builder()
+            .bounds(0, 90)
+            .withIncrement(5.0)
+            .withValue(0.0).build();
+
+    @Override
+    public void initialize() {
+        hoodMotor = new WPI_TalonSRX(RobotConstants.CAN.HOOD_MOTOR);
+
+        hoodMotorController = new TalonPositionController(hoodMotor, frc.robot.RobotConstants.MOTOR_SETTINGS.HOOD, false);
+        hoodMotorController.configure();
+        hoodMotor.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
```

Comment:
You do not need to configure limit switches anymore... it has been integrated into the talon settings object

Reply:
cool